### DISCUSSION TITLE
Can't specify minimum delay when using `retry_backoff` with jitter

### DISCUSSION THREAD

**Discussion by bendavis78** [celery/celery#8503] [Q&A]
From the docs for `retry_backoff`:

> If this option is set to a number, it is used as a delay factor. For example, if this option is set to 3, the first retry will delay 3 seconds, the second will delay 6 seconds, the third will delay 12 seconds, the fourth will delay 24 seconds, and so on. 

I understand jitter is added by default, but the above implies the first retry delay will be the number given in `retry_backoff`. However, with the current behavior, the first retry the countdown is often below the given`retry_backoff` number, as you can see in the following test:

```python
In [41]: for test in range(0, 5):
    ...:     print('---')
    ...:     for retry_num in range(0, 5):
    ...:         countdown = get_exponential_backoff_interval(5, retry_num, 180, full_jitter=True)
    ...:         print(countdown)

---
3
4
9
9
69
---
3
0
10
8
77
---
3
8
20
39
65
---
1
6
15
20
70
---
0
0
8
17
52
```

Is this expected behavior? It seems like we should have some way to control the minimum, but right now that doesn't seem to be possible.

**Comment by jogwen** [comment]
I agree, I'd like to be able to do something like ensure a third party API  gets a whole hour to recover before we give up trying it (maybe to ride out rate limiting). I can do that by turning off jitter, but jitter is a good thing, I'd just like to be able to specify the minimum for jitter so that it chooses a random number in range(retry_jitter_min, retry_backoff+1) rather than range(0, retry_backoff+1).

**Comment by chris-benninger-sophos** [comment]
To add to this, I was also surprised to find this behaviour and that the `default_retry_delay` was not the floor but is in-fact ignored altogether. I was expecting something like:   `range(default_retry_delay, default_retry_delay + retry_backoff)`
